infra/aws/cloudformation/iam/cdn.packages.k8s.io/cloudformation.yaml

```diff
@@ -0,0 +1,210 @@
+Resources:
+  #########################################
+  # Terraform State                       #
+  #########################################
+
+  # TFStateBucket (packages-k8s-io-tfstate) is S3 bucket for storing Terraform
+  # state
+  TFStateBucket:
+    Type: AWS::S3::Bucket
+    # We retain the bucket to ensure that accidental deletion of the stack
+    # doesn't destroy Terraform state
+    DeletionPolicy: Retain
+    Properties:
+      BucketName: cdn-packages-k8s-io-tfstate
+      AccessControl: Private
```

Comment:
I'd turn off ACLs and use a bucket policy instead.

Reply:
I think that we can just drop this. The bucket should be private by default.

Reply:
After dropping ACL:

![image](https://github.com/kubernetes/k8s.io/assets/18719127/1705f5d8-55c2-4561-8433-9f9aba5efbe5)